### ISSUE TITLE
Missing a line

### ISSUE THREAD

**Issue by agent2ouble** [home-assistant/home-assistant.github.io#34357]
### Feedback

Missing this:

password: YOUR_PASSWORD

In:

# Example configuration.yaml entry for Netgear device
device_tracker:
  - platform: netgear
    host: IP_ADDRESS
    username: YOUR_USERNAME
    interval_seconds: 10
    consider_home: 180
    new_device_defaults:
      track_new_devices: true

### URL

https://www.home-assistant.io/integrations/device_tracker/

### Version

2024.8.2

**Comment by agent2ouble** [comment]
This is still not fixed in the documentation.

**Comment by agent2ouble** [comment]
This is still not fixed in the documentation.

**Comment by agent2ouble** [comment]
Six months later. Still not fixed. So easy to do.

**Comment by MartinHjelmare** [comment]
Why does that matter? It's just an example. It's not documenting the netgear integration.

If you think it's important please open a pull request and change it.

I'll close here since there's nothing wrong.

**Comment by agent2ouble** [comment]
It’s part of documentation, where accuracy matters. What’s the point of giving an incomplete example?

**Comment by MartinHjelmare** [comment]
How do you mean incomplete? How do you know that password should be included? It's not a real integration where we know what items are required. It's an example which focuses on the use of `interval_seconds` and `consider_home`.

**Comment by agent2ouble** [comment]
It's incomplete because password: YOUR_PASSWORD is used in the first YAML example under Configuring a device_tracker platform, but not in the second example.

Example 1
# Example configuration.yaml entry for Netgear device
device_tracker:
  - platform: netgear
    host: IP_ADDRESS
    username: YOUR_USERNAME
    password: YOUR_PASSWORD
    new_device_defaults:
      track_new_devices: true

Example 2
# Example configuration.yaml entry for Netgear device
device_tracker:
  - platform: netgear
    host: IP_ADDRESS
    username: YOUR_USERNAME
    interval_seconds: 10
    consider_home: 180
    new_device_defaults:
      track_new_devices: true

Like you said, I guess it doesn't matter because it's just an example. But it's still inconsistent and easy to fix.

**Comment by MartinHjelmare** [comment]
Yes, it's easy to change. You're welcome to make a pull request.
